src/main/java/edu/harvard/iq/dataverse/search/SolrSearchResult.java

```diff
@@ -901,12 +901,10 @@ public void setReleaseOrCreateDate(Date releaseOrCreateDate) {
     }
 
     public String getDateToDisplayOnCard() {
-        return DateUtil.formatDate(dateToDisplayOnCard,"MMM dd, yyyy");
```

Comment:
@JayanthyChengan is this method still used anywhere? (since you changed line 420); if so it would make more send to have line 420 call this, so if we have to change we only have to change in one place; but if that's it's only use, we don't even need this)

Reply:
This method is used in search-include-fragment.xhtml
          Line 464:              <h:outputText value="#{result.dateToDisplayOnCard}" styleClass="text-muted" style="margin-right:.5em;"/>
          Line 522:              <h:outputText styleClass="text-muted" value="#{result.dateToDisplayOnCard}"/>
          Line 590:                  <h:outputText styleClass="text-muted" value="#{result.dateToDisplayOnCard} - "/>

Reply:
yes, right. (I saw it in the json and forgot that of course it's also on the page) Solet's change the json call back to using this message, so that the logic is only defined once.